src/components/@shared/AvailableNetworks/index.module.css

```diff
@@ -0,0 +1,12 @@
+.alert {
+  margin-bottom: 2rem;
+}
+.networks {
+  column-count: 2;
```

Comment:
you are assuming that this component will always be a direct children of an element which has `display: grid` on it which goes against the isolated components approach all components under `@shared` should follow. And hardcoded message also assumes component is only used within publish.

On first quick look, it seems the network list alone should be the `<AvailableNetworks />` component, with `display: grid` set on its root element, detached from the message which then would be added whenever component is used, or pass `message` as an optional prop

Reply:
I'm not sure I follow what you're saying here? This isn't a grid, the parent component is just the default `display: block`

Reply:
yeah, my bad, `column-count` indeed has nothing to do with `display: grid` and we cleaned up markup a lot already. Still, I don't see this component being used outside of `src/components/Publish` so tending towards moving it there as it is simply not a shared component, and its messaging is only for publishing. As said, we could move the message out of component and pass it as prop but that all feels like premature optimization which we can do once there's a need using this component outside of publish.

Reply:
Ok cool, I'll move it into `src/components/Publish` for now